core/utils/rect.ts

```diff
@@ -18,6 +18,9 @@ goog.declareModuleId('Blockly.utils.Rect');
  * Class for representing rectangular regions.
  */
 export class Rect {
+  public height;
```

Comment:
you don't need `public` since that's the default.

also, these only ever being set at construction is slightly weird since the other properties are also public and could be changed after the rectangle is constructed, and then the height and width would be incorrect 

and we do sometimes change them after construction e.g. https://github.com/google/blockly/blob/develop/core/workspace_svg.ts#L1901

so I get why you added them as a convenience but I wonder if they would be better as methods than properties, or maybe this is a good use case for a getter (though the style guide notes that transpiling getters results in bloated code)

Reply:
I'm going to change them to get-methods.